package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@google-cloud/vmmigration",
```

Comment:
we've been doing kebab case for distribution names and library names, but I think it's too late to change the repo name now so probably best to leave it as is, just FYI!

Reply:
I noticed that PHP and Java also opted for keeping `vmmigration`, my preference would be for keeping the name, since it reduces manual changes.